server/src/main/java/au/org/aodn/ogcapi/server/common/RestApi.java

```diff
@@ -136,6 +136,8 @@ public ResponseEntity<ConfClasses> getConformanceDeclaration(String f) {
             },
             method = RequestMethod.GET)
     public ResponseEntity<?> getCollections(
+            @Parameter(in = ParameterIn.QUERY, description = "Property to be return" ,schema=@Schema())
+                @Valid @RequestParam(value = "property", required = false) List<String> property,
```

Comment:
if it will be a list of properties, I think naming the param `properties` will be clearer.

Reply:
OGC call it property, so I just follow

Reply:
ok then